An error on osx-64:
```
  ./astropy/io/fits/src/compressionmodule.c:100:10: fatal error: 'numpy/arrayobject.h' file not found

  #include <numpy/arrayobject.h>

           ^~~~~~~~~~~~~~~~~~~~~

  1 warning and 1 error generated.

  error: command '/var/folders/sy/f16zz6x50xz3113nwtb9bvq00000gp/T/abs_croot-gstm8z3r/astropy_1647505681016/_build_env/bin/x86_64-apple-darwin13.4.0-clang' failed with exit code 1

  ERROR: Failed building wheel for astropy

ERROR: Could not build wheels for astropy which use PEP 517 and cannot be installed directly
```

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -28,6 +28,7 @@ build:
 requirements:
   build:
     - {{ compiler('c') }}
+    - pkg-config  # [osx and x86_64]
   host:
     - python 
     - cython >=0.29.22
@@ -41,8 +42,7 @@ requirements:
     - wheel
   run:
     - python
-    - {{ pin_compatible("numpy") }} # [not (osx and x86_64)]
-    - numpy >=1.19  # [(osx and x86_64) and py==39]
+    - {{ pin_compatible("numpy") }}
     - importlib-metadata
     - packaging >=19.0
     - pyerfa >=2.0

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -28,7 +28,7 @@ build:
 requirements:
   build:
     - {{ compiler('c') }}
-    - pkg-config  # [osx and x86_64]
+    - pkg-config
   host:
     - python 
     - cython >=0.29.22

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -41,7 +41,8 @@ requirements:
     - wheel
   run:
     - python
-    - {{ pin_compatible("numpy") }}
+    - {{ pin_compatible("numpy") }} # [not (osx and x86_64)]
+    - numpy >=1.19  # [(osx and x86_64) and py==39]
     - importlib-metadata
     - packaging >=19.0
     - pyerfa >=2.0

